src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,20 @@
+package lotto.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private final Scanner scanner = new Scanner(System.in);
+    private static final String INSERT_MONEY_MESSAGE = "구입금액을 입력해 주세요.";
+    private static final String WINNER_NUMBER_INPUT_MESSAGE = "\n지난 주 당첨 번호를 입력해 주세요.";
```

Comment:
```suggestion
    private static final String INSERT_MONEY_MESSAGE = "구입금액을 입력해 주세요.";
    private static final String WINNER_NUMBER_INPUT_MESSAGE = "\n지난 주 당첨 번호를 입력해 주세요.";
    private static final Scanner scanner = new Scanner(System.in);
```

일반적으로 자바 코드 컨벤션에서는 클래스 변수가 일반 변수보다 상위에 위치하는게 맞기도 해서 참고하시면 좋을 것 같아요!
![스크린샷 2022-05-17 오전 12 37 16](https://user-images.githubusercontent.com/31276206/168630626-289ac43c-f9f9-4eae-821b-50692d70d2e8.png)

추가로 scanner도 InputView 객체가 생성될때마다 같이 생성될 필요는 없어서 클래스 변수로 선언해줘도 좋을 것 같네요!